server/src/domain/auth/auth.service.ts

```diff
@@ -274,8 +273,7 @@ export class AuthService {
       }
 
       user = await this.userCore.createUser({
-        firstName: profile.given_name || '',
-        lastName: profile.family_name || '',
+        name: `${profile.given_name || ''} ${profile.family_name || ''}`,
```

Comment:
I think there is a name on the profile or full name. We should use that first and then fall back to what your have here.